.env.stable

```diff
@@ -3,7 +3,7 @@
 #
 
 
-PFCP_AGENT_IMAGE=omecproject/upf-epc-pfcpiface:master-2f04e3a
```

Comment:
do we have to revert this before merging ?

Reply:
Yes, we need first to merge the change in pfcp-agent.

Reply:
All the prs have been merged in upf. Should we revert this ?